sycl/test-e2e/Matrix/SG32/joint_matrix_half_accumulator.cpp

```diff
@@ -0,0 +1,37 @@
+//==-------SG32/joint_matrix_half_accumulator.cpp  - DPC++ joint_matrix ----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: gpu-intel-dg2
+// UNSUPPORTED-INTENDED: SG size = 32 is not supported for SYCL Joint Matrix on
+// DG2
+
```

Comment:
Different C and D types are not supported on the CPU. So we need to add `unsupported` comment to the two accumulator tests as follows:
```
// UNSUPPORTED: cpu
// UNSUPPORTED-INTENDED: Different C and D types are not supported on AMX
```